src/tests/test_eigencuda.cc

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
rename test file. I would prefer two different files, one for CUDA matrix and one for the Pipeline